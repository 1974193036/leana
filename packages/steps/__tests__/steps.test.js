import ssyySteps from "../src/steps.vue"
import { mount } from "@vue/test-utils"

describe('ssyy-steps', () => {
  test('ssyy-steps-snapshot', () => {
    const wrapper = mount(ssyySteps, {
      propsData: {
        count: 4,
        active: 1
      }
    })
    expect(wrapper.vm.$el).toMatchSnapshot() // 生成快照
  })
})