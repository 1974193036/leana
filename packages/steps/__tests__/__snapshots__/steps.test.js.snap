// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ssyy-steps ssyy-steps-snapshot 1`] = `
<div
  class="lg-steps"
>
  <div
    class="lg-step-item lg-step-item-finshed"
  >
    <div
      class="lg-step"
    >
      1
    </div>
  </div>
  <div
    class="lg-step-item lg-step-item-actived"
  >
    <div
      class="lg-step"
    >
      2
    </div>
  </div>
  <div
    class="lg-step-item lg-step-item-wait"
  >
    <div
      class="lg-step"
    >
      3
    </div>
  </div>
  <div
    class="lg-step-item-last lg-step-item-wait"
  >
    <div
      class="lg-step"
    >
      4
    </div>
  </div>
</div>
`;
